run/README.namelist

```diff
@@ -161,117 +161,117 @@ For automatic moving nests: requires special input data, and environment variabl
- zap_close_levels                    = 500      ; ignore isobaric level above surface if delta p (Pa) < zap_close_levels
- lowest_lev_from_sfc                 = .false.  ; place the surface value into the lowest eta location
-                                                ; T = use surface value as lowest eta (u,v,t,q)
-                                                ; F = use traditional interpolation
- force_sfc_in_vinterp                = 1        ; use the surface level as the lower boundary when interpolating
-                                                ; through this many eta levels
-                                                ; 0 = perform traditional trapping interpolation
-                                                ; n = first n eta levels directly use surface level
- maxw_horiz_pres_diff                = 5000     ; Pressure threshold (Pa).  For using the level of max winds, when the
-                                                ; pressure difference between neighboring values exceeds this maximum,
-                                                ; the variable is NOT inserted into the column for vertical interpolation.
-                                                ; ARW real only.
- trop_horiz_pres_diff                = 5000     ; Pressure threshold (Pa).  For using the tropopause level, when the
-                                                ; pressure difference between neighboring values exceeds this maximum,
-                                                ; the variable is NOT inserted into the column for vertical interpolation.
-                                                ; ARW real only.
- maxw_above_this_level               = 30000    ; Minimum height (actually it is pressure in Pa) to allow using the 
-                                                ; level of max wind information in real.  With a value of 300 hPa, then
-                                                ; a max wind value at 500 hPa will be ignored.
-                                                ; ARW real only.
- use_maxw_level                                 ; 0=do not use max wind speed level in vertical interpolation inside
-                                                ; of the ARW real program, 1 = use level
- use_trop_level                                 ; as above, with tropopause level data
- sfcp_to_sfcp                        = .false.  ; Optional method to compute model's surface pressure when incoming
-                                                ; data only has surface pressure and terrain, but not SLP
- smooth_cg_topo	                     = .false.  ; Smooth the outer rows and columns of domain 1's topography w.r.t.
-                                                ; the input data
- use_tavg_for_tsk                    = .false.  ; whether to use diurnally averaged surface temp as skin temp. The 
+ num_metgrid_soil_levels             = 4        ! number of vertical soil levels or layers input
```

Comment:
@smileMchen 
Ming,
In this section, would you add the missing "!" characters back in? The original README.namelist had the ";" character.